### ISSUE TITLE
Create license for usage

### ISSUE THREAD

**Issue by peter-walter** [visual-framework/vf-core#1142]
Maybe Apache 2.0 as one of the most permissive licenses.

**Comment by khawkins98** [comment]
Adding some helpful background info that @pwalter-ebi dug up earlier:

> Found the attached email regarding licenses which was discussed at Web Content Committee Feb 2018
> https://www.ebi.ac.uk/seqdb/confluence/display/WCC/2018-02-09+Meeting+notes
> 
> and Web Guidelines Committe March 2018
> https://www.ebi.ac.uk/seqdb/confluence/display/WGC/2018-03-06+WGC+Meeting+Minutes
> 
> The issue is still open in Gitlab
> https://github.com/ebiwd/EBI-Framework/issues/125

@sturobson Any opinion to _not_ use Apache 2.0?

**Comment by sturobson** [comment]
I'm flabbergasted that we got to today without noticing the lack of one :sob:

I think 2.0 should be good to go -- I think we'd need to check how that works with any other things that are used to make sure we're not breaking their licences(?) - learn, fractal, Wordpress. 11ty

**Comment by khawkins98** [comment]
tl;dr: Should be good on Apache 2.0

---

https://www.apache.org/foundation/license-faq.html#GPL 

> IS THE APACHE LICENSE COMPATIBLE WITH THE GPL (GNU PUBLIC LICENSE)?
> From the Free Software Foundation website:
> 
> Apache License, Version 2.0
> 
> This is a free software license, compatible with version 3 of the GPL. Please note that this license is not compatible with GPL version 2, because it has some requirements that are not in that older version. These include certain patent termination and indemnification provisions.
> 

Also compatible with MIT licensed things: https://exygy.com/blog/which-license-should-i-use-mit-vs-apache-vs-gpl/

- 11ty mit: https://github.com/11ty/eleventy/blob/master/LICENSE
- fractal mit: https://github.com/FractalBlocks/Fractal.js/blob/master/LICENSE
- lerna mit: https://github.com/lerna/lerna/blob/master/LICENSE

☝️  But I don't think it should matter too much as we're not shipping this code, only using it in the dynamic build (just in the same way we run on a website on a Linux server)

**Comment by peter-walter** [comment]
NPM seem to think it is Apache 2.0 already:

![image](https://user-images.githubusercontent.com/13538087/94810848-7d0b7480-03ec-11eb-827b-1c173a90a3ed.png)

**Comment by khawkins98** [comment]
Ah, yes, I knew we had already talked about this. 

https://github.com/visual-framework/vf-core/blob/8b52cf5515d30445f97bf9983d34051f51a48fac/package.json#L41

But should still add license file to the repo.

**Comment by khawkins98** [comment]
Footnote link: https://www.quora.com/I-have-an-Apache-2-0-Licensed-project-and-I-would-like-to-use-MIT-licensed-components-What-are-my-sublicensing-relicensing-obligations
